### DISCUSSION TITLE
上周续购了一年专业版，今天变为免费版，使用邮箱登录一直无法收到验证码

### DISCUSSION THREAD

**Discussion by zxx1025** [getlantern/lantern-forum-cn#791] [Q&A]
如题，账号：zxx1025@qq.com
![IMG_7514](https://github.com/user-attachments/assets/da3d46c2-532c-49d7-bacd-0195d05ec93c)

如确实无法发送国内邮箱，可替换邮箱 zxx1025@gmail.com

**Comment by SLS321** [comment]
蓝灯工作人员跑路了吧，我也遇到一样的问题，发送各种反馈都没回应~

**Reply by MaxMKMax** [reply]
@SLS321 您好. 请问您是遇到什么问题? 我能怎么帮您呢. 如果你们有问题最好的方法是开自己的帖子.
如果您在别人的帖子留言很容易混在一起或漏掉的. 您重新开个帖子然后附上支付单据和账号邮箱. 我帮您提交问题上去. 谢谢您的耐心等待.

**Comment by zxx1025** [comment]
@Mike777Khoo 请帮我解决

**Reply by MaxMKMax** [reply]
@zxx1025 
您好. 已经帮您提交问题上去了. 谢谢您支持蓝灯!

**Comment by SLS321** [comment]
好的，谢谢你的提醒.

Mike K ***@***.***> 于 2024年8月20日周二 01:43写道：
